index.bs

```diff
@@ -2216,6 +2218,46 @@ The primary difference between {{XRViewerPose}} and {{XRPose}} is the inclusion
+  - Each point in the {{XRBoundedReferenceSpace/boundsGeometry}} must be [=limiting|limited=] to a reasonable distance from the reference space's [=native origin=]; the suggested distance is 15 meters in all directions.
+  - Each point in the {{XRBoundedReferenceSpace/boundsGeometry}} must be [=rounding|rounded=] sufficiently to prevent fingerprinting. Rounding to the nearest 5cm is suggested. For user's safety, rounded points values MUST NOT fall outside the original bounds.
+  - The <code>Y</code> value of the [=native origin=] must be [=rounding|rounded=] sufficiently to prevent fingerprinting of lower-order bits; rounding to the nearest 1cm is suggested.
+  - All poses returned relative to a {{XRReferenceSpaceType/"bounded-floor"}} reference space MUST be [=limiting|limited=] to a reasonable distance beyond the {{XRBoundedReferenceSpace/boundsGeometry}} in all directions; the suggested distance is 1 meter beyond the bounds in all directions.
+
+#### local-floor #### {#local-floor-privacy}
+On devices which support [=6DoF=] tracking, {{XRReferenceSpaceType/"local-floor"}} reference spaces may be used to perform gait analysis, allowing user profiling and fingerprinting. In addition, because the {{XRReferenceSpaceType/"local-floor"}} reference spaces provide an established floor level, it may be possible for a site to infer the user's height, allowing user profiling and fingerprinting.
+
+If {{XRReferenceSpaceType/"local-floor"}} is included in an {{XRSession}}'s [=requested features=] the user agent MUST ensure [=user intent=] to allow {{XRReferenceSpaceType/"local-floor"}} tracking is well understood, either via [=explicit consent=] or [=implicit consent=], or the feature MUST be rejected.
+
+Any {{XRReferenceSpaceType/"local-floor"}} reference spaces MUST respect the following restrictions:
+  - If the floor level is based on sensor data or is set to a non-default emulated value, the <code>Y</code> value of the [=native origin=] must be [=rounding|rounded=] sufficiently to prevent fingerprinting of lower-order bits; rounding to the nearest 1cm is suggested.
+  - All poses returned relative to a {{XRReferenceSpaceType/"local-floor"}} reference space MUST be [=limiting|limited=] to a reasonable distance from the reference space's [=native origin=]; the suggested distance is 15 meters in all directions.
+
+#### local #### {#local-privacy}
+On devices which support [=6DoF=] tracking, {{XRReferenceSpaceType/"local"}} reference spaces may be used to perform gait analysis, allowing user profiling and fingerprinting.
+
+If {{XRReferenceSpaceType/"local"}} is included in an {{XRSession}}'s [=requested features=] and the {{XRSession}}'s [=XRSession/mode=] is {{XRSessionMode/"inline"}}, the user agent MUST ensure [=user intent=] to allow {{XRReferenceSpaceType/"local"}} tracking is well understood, either via [=explicit consent=] or [=implicit consent=], or the feature MUST be rejected.
```

Comment:
Note: User intent is also required for 'local' for immersive sessions; the difference is that the feature is added implicitly. That doesn't appear to be covered here or in the session creation section.

I wonder if that needs an extra step in the session creation algorithm above, something like:

> When creating any immersive session on a device that supports 6DOF tracking, the user agent MUST ensure [=user intent=] to allow {{XRReferenceSpaceType/"local"}} tracking is well understood, either via [=explicit consent=] or [=implicit consent=], or the feature MUST be rejected.

Reply:
Adjusted this text to account for that and added a non-normative note reminding the reader that this is always included as a default feature of immersive sessions.

Reply:
> If you have any thoughts on how to ease that tension, I'm happy to hear it!